Makefile.template

```diff
@@ -0,0 +1,13 @@
+# For scripted languages it might be unnecessary to change anything except the line below `run:`
```

Comment:
Neither is it my native language, which is obvious from my reading mistake :)
It's clear from the context, no need to change anything. I just thought there was a mistake.